packages/offix-conflicts-server/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "offix-server",
-  "version": "0.6.0",
+  "name": "offix-conflicts-server",
```

Comment:
perhaps `offix-server-conflicts?` and then we will be able to release other packages in the future with the pattern `offix-server-<feature>` if necessary?